docs/tools-resources/faq.mdx

```diff
@@ -113,4 +113,13 @@ Yes. It is only optional, when there is no governance policy. When there is a go
 
 ### 17. Are the governance voting threshold Parameters set to be fixed at 2 decimal places?
 
-No, they are rational numbers on the unit interval where both numerator and denominator are specified by 64bit unsigned integers. So, those thresholds can be very precise.
\ No newline at end of file
+No, they are rational numbers on the unit interval where both numerator and denominator are specified by 64bit unsigned integers. So, those thresholds can be very precise.
+
+### 18. Do unwithdrawn staking rewards count towards one's voting power/ is counted in active/inactive voting stake amounts?
+
+Yes, the amount in the reward account is counted towards active stake, which means it will also participate in the voting.
+This decision was to prevent people from constantly withdrawing rewards just so they can get rewards on that amount.
```

Comment:
```suggestion
The amount in the reward account is considered part of the active stake, thus participating in the voting process. This decision aims to prevent individuals from repeatedly withdrawing rewards solely to earn rewards on that amount.
```